.circleci/config.yml

```diff
@@ -145,12 +145,13 @@ jobs:
 
             # MNIST tests
 
-            # 0) mnist_patch.py
+            # 0) download MNIST
             # https://github.com/pytorch/ignite/issues/1737
-            export install_wget_cmd='apt-get update && apt-get install -y wget'
-            docker exec -it pthd /bin/bash -c "$install_wget_cmd"
-            export minst0_cmd='CUDA_VISIBLE_DEVICES=0 python examples/mnist/mnist_patch.py'
-            docker exec -it pthd /bin/bash -c "$minst0_cmd"
+            export raw_mnist_dir='./MNIST/raw'
+            export download_mnist_cmd='git clone https://github.com/pytorch-ignite/download-mnist-github-action.git $raw_mnist_dir'
```

Comment:
I think you should use `"` instead `'` if we'd like to resolve vars:
```
export raw_mnist_dir='./MNIST/raw'
export download_mnist_cmd='git clone https://github.com/pytorch-ignite/download-mnist-github-action.git $raw_mnist_dir'
echo $download_mnist_cmd
> git clone https://github.com/pytorch-ignite/download-mnist-github-action.git $raw_mnist_dir

```

Reply:
Thanks! I'll change that after action update